package.json

```
@@ -64,9 +64,11 @@
   "devDependencies": {
     "@babel/cli": "^7.0.0",
     "@babel/core": "^7.0.0",
+    "@rollup/plugin-babel": "^5.2.1",
+    "@rollup/plugin-commonjs": "^15.1.0",
+    "@rollup/plugin-node-resolve": "^9.0.0",
```

```suggestion
    "@rollup/plugin-babel": "^5.0.0",
    "@rollup/plugin-commonjs": "^15.0.0",
    "@rollup/plugin-node-resolve": "^9.0.0",
```

Assuming we’re not using any fancy new features, zero-filling will reduce some noice in the future